engine/src/scheme.rs

```diff
@@ -904,6 +936,10 @@ impl<'s> Scheme {
             index,
         })
     }
+
+    pub(crate) fn get_nil_not_equal_behavior(&self) -> bool {
```

Comment:
nit: Should we drop the `get_` prefix?

Reply:
Don't have a strong opinion, it's a private method anyway so it doesn't matter much